form.yml.erb

```diff
@@ -1,166 +1,98 @@
-  label: "Nextflow Pipeline Version"
-  help: >
-    Leave blank to run the latest. 
-    If you are running an nf-core pipeline, you have the option of using `dev`. 
-    It's highly recommended to use versioned releases of pipelines.
+  inputs:
+    label: WDL Workflow Inputs
+    help: Inputs can be either paths to JSON or YAML files containing inputs or key-value pairs passed in on the command line to Sprocket.
+    data-filepicker: true
+    data-target-file-type: files
+    readonly: false
 
-  run_uid:
-    widget: "text_field"
-    label: "Run Unique Name"
-    help: |
-      The run's name should be unique and must be reused to resume running a pipeline.
-  
-  custom_config:
-    label: "Custom Configuration File"
+  entrypoint:
+    label: Entrypoint
+    help: The name of WDL workflow or task to run; required if `inputs` is empty.
     data-filepicker: true
-    data-target-file-type: files # Valid values are: files, dirs, or both
+    data-target-file-type: files
     readonly: false
-    help: "Select your custom configuration file; leave blank to use the default (recommended unless you know what you're doing)."
 
-  nextflow_params:
-    label: "Nextflow Params File"
+  sprocket_config:
+    label: Sprocket Configuration File
+    help: Select your custom Sprocket configuration file (leave blank to use the default).
     data-filepicker: true
-    data-target-file-type: files # Valid values are: files, dirs, or both
+    data-target-file-type: files
     readonly: false
-    help: "Select your parameters file (JSON or YAML)."
 
+  max_scatter_concurrency:
+    label: Max Scatter Concurrency
+    help: |
+      The maximum number of subtasks each `scatter` will try executing at once. This is *not* a direct
+      limit on the total number of concurrent tasks, but can affect the number of jobs that get queued
+      at one time.
+    widget: number_field
+    value: 100
+    min: 1
+    max: 1000
+
+  apptainer_images_dir:
+    label: Apptainer Images Cache
+    value: "~/.cache/sprocket-apptainer-images"
+    help: |
+      The directory where container `.sif` images will be cached (leave blank to use the default).
+      Be sure to regularly clear this directory out.
+    data-filepicker: true
+    data-target-file-type: dirs
+    readonly: false
 
+  bc_num_hours:
+    help: The maximum number of hours the submitted batch job may run.
 
-# All of the attributes that make up the Dashboard form (in respective order),
-# and made available to the submit configuration file and the template ERB
-# files
-#
-# @note You typically do not need to modify this unless you want to add a new
-#   configurable value
-# @note If an attribute listed below is hard-coded above in the `attributes`
-#   option, then it will not appear in the form page that the user sees in the
-#   Dashboard
 form:
   - modules
-  - choose_queue
-  - pipeline_name
-  - version
-  - run_uid
+  - queue
+  - workflow
+  - inputs
+  - entrypoint
+  # - sprocket_config
```

Comment:
```suggestion
  # Currently this is disabled because we can't explicitly specify more than one configuration file; will need a change to `sprocket`.
  # - sprocket_config
```